### ISSUE TITLE
Home Assistant integration: device class NONE for sensor.disk_usage

### ISSUE THREAD

**Issue by ocean75** [bkbilly/lnxlink#45]
Hello,

Since updating Home Assistant to 2023.3 I am receiving the following error message in the HA log file. I assume it has something to do with the MQTT message, but unsure how to correct the issue. Thank you.

```txt
Sensor sensor.disk_usage_ubuntu has device class None, state class None and unit thus indicating it has a numeric value; however, it has the non-numeric value: False (<class 'str'>); Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+mqtt%22
```

**Comment by bkbilly** [comment]
Thank you for pointing this out because I couldn't understand why all the sensors were numeric.
I'll fix it on the next release.

**Comment by bkbilly** [comment]
I've released a new version 2023.3.0, please let me know if this fixes your issue.

**Comment by ocean75** [comment]
I believe it did the trick. I am not receiving the same error message anymore. Thank you.
